docs/standard/microservices-architecture/net-core-net-framework-containers/net-framework-container-scenarios.md

```diff
@@ -53,8 +53,8 @@ In the meantime, if any platform or service in Azure still doesn't support .NET
 - **Porting from .NET Framework to .NET Core**  
     [https://docs.microsoft.com/dotnet/core/porting/index](../../../core/porting/index.md)
 
-- **.NET Framework on Docker Guide**  
-    [https://docs.microsoft.com/dotnet/framework/docker/](../../../framework/docker/index.md)
+- **.NET Core on Docker Guide**
+    [https://docs.microsoft.com/dotnet/framework/docker/](../../../core/docker/index.md)
```

Comment:
The visible part of this link still shows "framework" instead of core
```suggestion
    [https://docs.microsoft.com/dotnet/core/docker/intro-net-docker](../../../core/docker/index.md)
```

Reply:
I've made the suggested change and also replaced `index.md` with `intro-net-docker.md`.